### PR TITLE
update developer guide

### PR DESCRIPTION
**Pull request by FeynmanZhou**

### CODE REVIEW

File: docs/CLI/5_developer_guide.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Developer Guide
 
+## Prerequisites
+
+- Go 1.18+
+- Git
+
+## Clone the repo
+
+You can clone the repo and get the main branch using the following command.
```
Comment (vsoch) [inline docs/CLI/5_developer_guide.md:10]:
```suggestion
You can clone the repo and get the main branch using the following command:
```

File: docs/CLI/5_developer_guide.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+
+```
```
Comment (vsoch) [inline docs/CLI/5_developer_guide.md:12]:
```suggestion
```bash
```

File: docs/CLI/index.md [hunk 3]
```diff
@@ -59,7 +51,7 @@ set PATH=%USERPROFILE%\bin\;%PATH%
 A public Docker image containing the CLI is available on [GitHub Container Registry](https://github.com/orgs/oras-project/packages/container/package/oras):
 
 ```
```
Comment (vsoch) [inline docs/CLI/index.md:53]:
```suggestion
```bash
```

**Review by vsoch — COMMENTED** [review]
A few nitpicks, but otherwise looks good!

**Review by SteveLasker — APPROVED** [review]
LGTM

File: docs/CLI/5_developer_guide.md [hunk 4]
```diff
@@ -1,5 +1,18 @@
 # Developer Guide
 
+## Prerequisites
+
+- Go 1.18+
+- Git
+
```
Comment (vsoch) [inline docs/CLI/5_developer_guide.md:7]: Nit: I would generally write out “repository” over “repo” so it errs on the side of more professional writing.

File: docs/CLI/5_developer_guide.md [hunk 5]
```diff
@@ -1,5 +1,18 @@
...
+## Clone the repo
+
+You can clone the repo and get the main branch using the following command:
+
+```bash
+https://github.com/oras-project/oras.git
```
Comment (vsoch) [inline docs/CLI/5_developer_guide.md:13]: Is there a git clone and cd missing here?

File: docs/CLI/index.md [hunk 6]
```diff
@@ -50,7 +50,7 @@ set PATH=%USERPROFILE%\bin\;%PATH%
 
 A public Docker image containing the CLI is available on [GitHub Container Registry](https://github.com/orgs/oras-project/packages/container/package/oras):
 
-```
+```bash
```
Comment (vsoch) [inline docs/CLI/index.md:53]: Hmm it’s definitely arduous to have to manually update the latest version in multiple places (the container tag here being one). That’s historically a drawback of mkdocs (over jekyll) that doesn’t support variables in the markdown. Perhaps looking into a plug-in might be an intermediate solution? https://github.com/rosscdh/mkdocs-markdownextradata-plugin

File: docs/CLI/5_developer_guide.md [hunk 7]
```diff
@@ -1,5 +1,24 @@
 # Developer Guide
 
+## Prerequisites
+
+- Go 1.18+
+- Git
+
+## Clone the repository
+
+You can clone the repo and get the main branch using the following command:
```
Comment (vsoch) [inline docs/CLI/5_developer_guide.md:10]:
```suggestion
You can clone the repository and get the main branch using the following command:
```

File: docs/CLI/5_developer_guide.md (hunk 4 above)
Comment (FeynmanZhou) [reply docs/CLI/5_developer_guide.md:7]: Agree.

File: docs/CLI/index.md (hunk 6 above)
Comment (FeynmanZhou) [reply docs/CLI/index.md]: @vsoch Good idea! I think we can raise another PR to introduce this plugin and then update those places in the whole documentation. I created a separate [issue](https://github.com/oras-project/oras-www/issues/47) to track it.

**Comment by FeynmanZhou** [comment]
@vsoch Thanks for your review. I updated it according to your suggestions.

**Review by vsoch — APPROVED** [review]
Nice work @FeynmanZhou!

File: docs/CLI/index.md (hunk 6 above)
Comment (vsoch) [reply docs/CLI/index.md]: Sounds good.

**Pull request merged**